examples/bidder/bidder.hpp

```diff
@@ -70,7 +70,7 @@ namespace vanilla {
             bid.id = boost::uuids::to_string(bidid); // TODO check documentation 
             // Is it the same as response.bidid?
             bid.impid = imp.id;
-            bid.price = ad->max_bid_micros / 1000000.0; // Not micros?
+            bid.price = ad->auth_bid_micros ? *ad->auth_bid_micros / 1000000.0 : ad->max_bid_micros / 1000000.0 ; // Not micros?
```

Comment:
can you please propose the function , like 
{code}
select_price(boost::optional<uint64_t> authorized_bid_micros , uint64_t max_bid_micros , double denom) {
   return authorized_bid_micros ? *authorized_bid_micros / denom : ad->max_bid_micros / denom;
}
{code}

Like that ?